highfidelity/hifi
# New model dialog fix

This fixes the custom query dialog box to allow for 3 lines of warning message instead of 2 so the message will not be cut off.

https://highfidelity.fogbugz.com/f/cases/12831/ADD-TO-WORLD-dialog-is-not-displaying-all-text-related-to-using-Exact-All-polygons-option

TESTING

- Open interface in desktop and go to your sandbox
- Open Menu>Edit>Asset Browser, select any fbx or obj file and "Add to World"
- A popup dialog box appears. Set automatic collisions to "Exact- All Polygons"
- A warning message should appear. It should be 3 lines long and not cut off: "Models with 'Exact' automatic collisions cannot be dynamic, and should not be used as floors"
- Open interface in HMD and go to your sandbox
- Open Tablet and Menu>Edit>Asset Browser, select any fbx or obj file and "Add to World"
- A popup dialog box appears. Set automatic collisions to "Exact- All Polygons"
- A warning message should appear. It should be 3 lines long and not cut off: "Models with 'Exact' automatic collisions cannot be dynamic, and should not be used as floors"
- Repeat test for Android...
